app/views/members/activities/show.html.haml

```diff
@@ -33,8 +33,4 @@
             - @reservists.each do |participant|
               = render 'members/activities/partials/participant_table_row', participant: participant, activity: @activity
 
-= link_to "mailto:pleasedotheneedful@svsticky.nl" , class: "btn btn-danger helpme"do
```

Comment:
This is not really part of this PR, but I guess a rebase will solve the problem

Reply:
Yep, fixed it!